CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing
+
+We love pull requests from everyone. Instructions:
+
+Fork, then clone the repo:
+
+    git clone git@github.com:your-username/bazel-deps.git
+
+Make sure the tests pass:
+
+    bazel test //...
+
+Make your change. Add tests for your change. Make the tests pass:
+
+    bazel test //...
+
+If your change affects the generated `workspace.bzl`, regenerate it:
+
+    bazel build src/scala/com/github/johnynek/bazel_deps:parseproject_deploy.jar
+    ./gen_maven_deps.sh generate -r $PWD \
+        -s 3rdparty/workspace.bzl -d dependencies.yaml
```

Comment:
you can now run 

`bazel run //:parse -- generate -r $PWD -s 3rdparty/workspace.bzl -d dependencies.yml`

Reply:
Yes, can we make the above change? It makes it faster to run since you don’t need to make the deploy jar.

Reply:
Done - that is better.